api/client/gro_client.py

```diff
@@ -257,19 +259,16 @@ def GDH(self, gdh_selection, **optional_selections):
 
         """
 
-        entity_keys = ['metric_id', 'item_id', 'region_id', 'partner_region_id',
-                       'source_id', 'frequency_id']
         entity_ids = [int(x) for x in gdh_selection.split('-')]
-        selection = dict(zip(entity_keys, entity_ids))
+        selection = dict(zip(DATA_SERIES_UNIQUE_COLS, entity_ids))
 
         # add optional pararms to selection
         for key, value in list(optional_selections.items()):
-            if key not in entity_keys:
+            if key not in DATA_SERIES_UNIQUE_COLS:
                 selection[key] = value
 
         self.add_single_data_series(selection)
-        df = self.get_df()
-        return df
+        return self.get_df(index_by_series=True).loc[[tuple(entity_ids)], :]
```

Comment:
this `.loc[[tuple(entity_ids)], :]` part will throw an error if given selection doesn't have at-level data. e.g client.GDH("860032-10009-1215-0-14-9")

Reply:
that's the intended behavior... gdh is expected to return only that series exactly.

Reply:
yeah I understand that - but it's showing 

`KeyError: "None of [MultiIndex([(860032, 10009, 1215, 0, 14, 9)],\n           names=['metric_id', 'item_id', 'region_id', 'partner_region_id', 'source_id', 'frequency_id'])] are in the [index]"`

should we return an empty data frame or something ?

Reply:
done